src/v/cloud_storage/topic_manifest.h

```diff
@@ -13,7 +13,7 @@
 #include "cloud_storage/base_manifest.h"
 #include "cloud_storage/fwd.h"
 #include "cloud_storage/types.h"
-#include "cluster/types.h"
+#include "cluster/topic_configuration.h"
```

Comment:
topic_configuration is what was used, but types.h is brining in a ton of other stuff.